content/news/2020-12-01-about-me.md

```diff
@@ -0,0 +1,115 @@
+title: All about my Outreachy internship with Mixxx DJ: Intro
+authors: Aanyu Deborah Oduman
+tags: personal, outreachy, internship
+date: 2020-12-01 23:15:00
+
+#### A little bit about me.
+
+ <!-- | [![hello]({static}/images/news/hello.jpg)]({static}/images/news/hello.jpg?) | [![deborah aanyu]({static}/images/news/deborah_intern.jpg)]({static}/images/news/deborah_intern.jpg) |
+|:---:|:---:| -->
+
+<!-- | hello | deborah aanyu | -->
```

Comment:
Please remove. If you want to add them again, these are preserved in the git history.

Reply:
Alright.